be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -387,7 +387,6 @@ Status HashJoinNode::close(RuntimeState* state) {
     }
 
     if (_shared_hashtable_controller) {
-        _shared_hashtable_controller->release_ref_count(state, id());
         _shared_hashtable_controller->wait_for_closable(state, id());
```

Comment:
Why there is still exist wait for closable here?

Reply:
Here wait for other instances to release the reference count(the other instances will release the ref count after they copied the shared resource).